src/mdl/isoscelles/isoquant_output_to_scanpy_seurat_input.py

```diff
@@ -0,0 +1,123 @@
+import csv
+from collections import defaultdict
+from pathlib import Path
+from typing import Sequence
+
+import h5py
+import numpy as np
+import scipy.io
+import sparse
+
+import pandas as pd
+import shutil
+import pysam
+import os
+import gzip
+
+import scanpy as sc
```

Comment:
Please use `black` and `ruff` to format the file (this will include sorting the imports). See [here](https://github.com/MethodsDev/mdl-core/blob/main/code-style.md) for more details (and also the config for this repo).